src/uikit.cs

```diff
@@ -21095,4 +21740,1026 @@ interface UIPointerShape : NSCopying {
+
+		[Export ("adjustsFontForContentSizeCategory")]
+		bool AdjustsFontForContentSizeCategory { get; set; }
+	}
+
+	interface UIPointerLockStateDidChangeEventArgs {
+		[NoWatch, NoTV, iOS (14,0)]
+		[NullAllowed]
+		[Export ("UIPointerLockStateSceneUserInfoKey")]
+		UIScene Scene { get; }
+	}
+
+	[NoWatch, NoTV, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIPointerLockState {
+
+		[Field ("UIPointerLockStateDidChangeNotification")]
+		[Notification (typeof (UIPointerLockStateDidChangeEventArgs))]
+		NSString DidChangeNotification { get; }
+
+		[Export ("locked")]
+		bool Locked { [Bind ("isLocked")] get; }
+	}
+
+	[NoWatch, NoTV, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UIScribbleInteraction : UIInteraction {
+
+		[Export ("initWithDelegate:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (IUIScribbleInteractionDelegate @delegate);
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		IUIScribbleInteractionDelegate Delegate { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; }
+
+		[Export ("handlingWriting")]
+		bool HandlingWriting { [Bind ("isHandlingWriting")] get; }
+
+		[Static]
+		[Export ("pencilInputExpected")]
+		bool PencilInputExpected { [Bind ("isPencilInputExpected")] get; }
+	}
+
+	interface IUIScribbleInteractionDelegate { }
+
+	[NoWatch, NoTV, iOS (14,0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface UIScribbleInteractionDelegate {
+
+		[Export ("scribbleInteraction:shouldBeginAtLocation:")]
+		bool ShouldBegin (UIScribbleInteraction interaction, CGPoint location);
+
+		[Export ("scribbleInteractionShouldDelayFocus:")]
+		bool ShouldDelayFocus (UIScribbleInteraction interaction);
+
+		[Export ("scribbleInteractionWillBeginWriting:")]
+		void WillBeginWriting (UIScribbleInteraction interaction);
+
+		[Export ("scribbleInteractionDidFinishWriting:")]
+		void DidFinishWriting (UIScribbleInteraction interaction);
+	}
+
+	interface IUISearchSuggestion { }
+
+	[TV (14,0), NoWatch, NoiOS]
+	[Protocol]
+	interface UISearchSuggestion {
+
+		[Abstract]
+		[NullAllowed, Export ("localizedSuggestion")]
+		string LocalizedSuggestion { get; }
+
+		[return: NullAllowed]
+		[Export ("localizedDescription")]
+		string GetLocalizedDescription ();
+
+		[return: NullAllowed]
+		[Export ("iconImage")]
+		UIImage GetIconImage ();
+	}
+
+	[NoWatch, TV (14,0), iOS (14,0)]
+	[Static, Partial]
+	interface UIConfigurationColorTransformer {
+
+		// Manually bound inside UIKit/UIConfigurationColorTransformer.cs
+		[Internal]
+		[Field ("UIConfigurationColorTransformerGrayscale")]
+		IntPtr _Grayscale { get; }
+	}
+
+	[TV (14,0), NoWatch, NoiOS]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UISearchSuggestionItem : UISearchSuggestion {
+
+		[Static]
+		[Export ("suggestionWithLocalizedSuggestion:")]
+		UISearchSuggestionItem Create (string suggestion);
```

Comment:
I wonder if localizedSuggestion is one of those strings that have to stay as an NSString so that they can be localized at runtime?

Reply:
better safe than sorry